experimental/FTheoryTools/test/FTM-1511-03209.jl

```diff
@@ -0,0 +1,7 @@
+h = literature_model(arxiv_id = "1511.03209")
+h_resolved = resolve(h, 1)
```

Comment:
Please move these two lines inside the testset. Otherwise, they will stay as globals for the rest of the test session and cannot get garbage collected